ProductivitySuite/docker_compose/intel/hpu/gaudi/compose.yaml

```diff
@@ -0,0 +1,376 @@
+# Copyright (C) 2024 Intel Corporation
```

Comment:
Please add a `test_compose_on_gaudi.sh` to cover the new code.

Reply:
like this one: https://github.com/opea-project/GenAIExamples/blob/53500d108d3075b7ced0faff8433095398480b2a/ProductivitySuite/tests/test_compose_on_xeon.sh

Reply:
Added it